packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -412,6 +413,112 @@ Content-Type: application/json;charset=UTF-8
+    4. Okta rejects the JWT if the `jti` claim is present and it has already been processed.
+
+#### Response properties
+
+| Property                  | Description                                                                                                         | Type    |
+| :------------------------ | :---------------------------------------------------------------------------------------------                      | :------ |
+| auth_req_id               | This is a unique identifier to identify the authentication request made by the Client.                              | String  |
+| interval                  | The minimum amount of time in seconds that the client should wait between polling requests to the token endpoint.   | Integer |
+| expires_in                | The expiration time of the `auth_req_id` in seconds.                                                                | Integer |
+
+#### Possible errors
+
+| Error Id                 | Details                                                                                                                                                                                                    |
+| :----------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| invalid_client           | The specified `client_id` isn't found.                                                                                                                                                                      |
+| invalid_request          | The request is missing a necessary parameter, the parameter has an invalid value, or the request contains duplicate parameters. |
+| invalid_scope            | The scopes list contains an invalid or unsupported value.                                                                                                                                                  |
+| unknown_user_id          | The OpenID Provider is not able to identify which end-user the Client wishes to be authenticated by means of the hint provided in the request.                                                                                                                                                  |
+| unauthorized_client      | The Client is not authorized to use this authentication flow.                                                                                                                                                  |
+| access_denied            | The server denied the request.                                                                                                                                                  |
+
+#### Request examples
+
```

Comment:
We add a sentence above each one to describe each so that users don't have to try and compare them to see what is different between them.

Also - question just for me - is it required to include either login_hint or id_token_hint in the request?

Reply:
yes, you need to pass either login_hint or id_token_hint in the request